powerstore/datasource_host_group_test.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 // Test to Fetch Host Groups
-func TestAccHostGroup_FetchHostGroup(t *testing.T) {
+func TestAccHostGroupDs_FetchHostGroup(t *testing.T) {
```

Comment:
Add the Ds here so that the tests can be run independently from the resource test.